www/litv00/du3_css_litv00.css

```diff
@@ -0,0 +1,106 @@
+body {
+	font-family: Helvetica, Arial, Sans-Serif;
+}
+.box1{
+	color: red;
+}
+.box2{
+	color: #00FF00;
+}
+.box3{
+	font-weight: bold;
+	color: rgb(0,0,255);
+}
+.box4{
+	font-size: 18px;
+}
+.box5{
+	font-size: 24px;
+	font-style: italic;
+}
+.box6{
+	color: white;
+	background-color: brown;
+}
+.box7{
+	width: 400px;
+	height: 300px;
+	border: 1px solid purple;
+	padding: 30px;
+}
+.box8{
+	width: 500px;
+	height: 200px;
+	background-color: red;
+	margin: 100px;
+}
+.box9{
+	display: none;
+}
+.box10{
+	width: 100%;
+	height: 150px;
+	background-color: blue;
+	padding: 50px;
+	border: 10px solid yellow;
+	box-sizing: border-box;
```

Comment:
very good, akorat to box-sizing: border-box dej do wildcard selektoru (hvezdicka) aby platila pro vsechno